### ISSUE TITLE
MSAC center metadata is throwing validation errors

### ISSUE THREAD

**Issue by floatplane** [NWACus/avy#469]
Most of `widget_config.stations` is missing, relative to what we expect
```bash
$ curl https://api.avalanche.org/v2/public/avalanche-center/MSAC | jq ".widget_config.stations"
{
  "token": "3cafafc1202aa2d56005eaa9d8ed8801bad2"
}
```

The schema expects much more
https://github.com/NWACus/avy/blob/30eee126b467a494b7a913af279d5bea2c17f420/types/nationalAvalancheCenter/schemas.ts#L355-L368

**Comment by floatplane** [comment]
damn, I think this is actually breaking the center selector :(

**Comment by floatplane** [comment]
fixed in #470
